app/views/management/index.html.erb

```diff
@@ -6,11 +6,11 @@
     <button type="button" class="btn btn-primary dropdown-toggle mr-3" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
       Index Metadata to Solr
     </button>
-    
+
     <div class="dropdown-menu">
-      <%= link_to "Index Voyager Records to Solr", metadata_source_path("ils"), class: "dropdown-item" %>
-      <%= link_to "Index Ladybird Records to Solr", metadata_source_path("ladybird"), class: "dropdown-item" %>
-      <%= link_to "Index ArchiveSpace Records to Solr", metadata_source_path("aspace"), class: "dropdown-item" %>
+      <%= link_to "Index Voyager Records to Solr", ms_path("ils"), class: "dropdown-item" %>
```

Comment:
why ms?

Reply:
Oh, this is because of trying to address the same issue as your first comment - conflicting routes when trying to create a new model. Can hopefully come up with a better solution.